src/components/molecules/actionables.tsx

```diff
@@ -0,0 +1,71 @@
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu"
+import clsx from "clsx"
+import React from "react"
+import Button from "../fundamentals/button"
+import MoreHorizontalIcon from "../fundamentals/icons/more-horizontal-icon"
+
+export type ActionType = {
+  label: string
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+  variant?: "normal" | "danger"
+  icon: React.ReactNode
+}
+
+type ActionablesProps = {
+  actions: ActionType[]
+}
+
+/**
+ * A component that accepts multiple actionables and renders them as a dropdown menu.
+ * If only a single actionable is provided, it will render a button instead.
+ */
+const Actionables: React.FC<ActionablesProps> = ({ actions }) => {
+  if (actions.length === 0) {
```

Comment:
Should check if actions is `undefined`, since they are not required
```suggestion
  if (actions?.length === 0) {
```